src/Integration.TeamExplorer.UnitTests/Integration.TeamExplorer.UnitTests.csproj

```diff
@@ -246,16 +246,7 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="ConnectSectionViewModelTests.cs" />
-    <Compile Include="Framework\ConfigurableTeamExplorer.cs" />
-    <Compile Include="Framework\ConfigurableTeamExplorerPage.cs" />
-    <Compile Include="HostedCommandControllerBaseTests.cs" />
-    <Compile Include="Progress\ProgressStepRunnerTests.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="SectionControllerTests.cs" />
-    <Compile Include="SonarQubeNavigationItemTests.cs" />
-    <Compile Include="SonarQubePageTests.cs" />
-    <Compile Include="TeamExplorerControllerTests.cs" />
+    <Compile Include="**\*.cs" Exclude="**\obj\**\*" />
```

Comment:
I can't say that's the easiest solution to work with but at least now this is consistent with how other projects are set up.

Reply:
I'd left it with explicit references because it was easier to work with, but it doesn't matter too much either way.